sdk/tables/Azure.Data.Tables/src/TableClient.cs

```diff
@@ -286,6 +288,7 @@ internal TableClient(Uri endpoint, string tableName, TableSharedKeyPipelinePolic
             Argument.AssertNotNullOrEmpty(tableName, nameof(tableName));
 
             _endpoint = endpoint;
+            SetEndpointStringWithoutTableName(_endpoint, tableName, out string endpointString, out _endpoint);
```

Comment:
_endpoint = GetEndpointStringWithoutTableName(_endpoint, tableName, out string endpointString)

Reply:
Do you use endpointString?

Reply:
nope - I was initially and factored it out.. Will remove.

Reply:
removed endpointString altogether and had it return _endpoint.